ibm_catalog.json

```diff
@@ -197,158 +208,34 @@
-                                "profile_name": "SLZ VSI Custom v0.2.0",
-                                "control_id": "IA-4(d)"
-                            },
-                            {
-                                "profile_name": "SLZ VSI Custom v0.2.0",
-                                "control_id": "IA-5(1)(a)"
-                            },
-                            {
-                                "profile_name": "SLZ VSI Custom v0.2.0",
-                                "control_id": "IA-5(1)(c)"
-                            },
-                            {
-                                "profile_name": "SLZ VSI Custom v0.2.0",
-                                "control_id": "IA-5(1)(d)"
-                            },
-                            {
-                                "profile_name": "SLZ VSI Custom v0.2.0",
-                                "control_id": "IA-5(1)(e)"
-                            },
-                            {
-                                "profile_name": "SLZ VSI Custom v0.2.0",
-                                "control_id": "IA-5(b)"
-                            },
-                            {
-                                "profile_name": "SLZ VSI Custom v0.2.0",
-                                "control_id": "IA-5(f)"
-                            },
-                            {
-                                "profile_name": "SLZ VSI Custom v0.2.0",
-                                "control_id": "IA-6"
-                            },
-                            {
-                                "profile_name": "SLZ VSI Custom v0.2.0",
-                                "control_id": "IA-2-0"
-                            },
-                            {
-                                "profile_name": "SLZ VSI Custom v0.2.0",
-                                "control_id": "SC-10"
-                            },
-                            {
-                                "profile_name": "SLZ VSI Custom v0.2.0",
-                                "control_id": "SC-11"
-                            },
-                            {
-                                "profile_name": "SLZ VSI Custom v0.2.0",
-                                "control_id": "SI-4(a)"
-                            },
-                            {
-                                "profile_name": "SLZ VSI Custom v0.2.0",
-                                "control_id": "SI-4(b)"
-                            },
-                            {
-                                "profile_name": "SLZ VSI Custom v0.2.0",
-                                "control_id": "SI-4(c)"
+                                "profile": {
+                                    "name": "IBM Cloud for Financial Services",
+                                    "version": "1.2.0"
+                                },
+                                "names": [
+                                    "AC-2(4)(0)",
```

Comment:
Ah the list was sorted alphabetically (instead of numerically) - fixing this now.
As for the removed controls, I just checked the reports, and it was a pagination issue. There was more passing controls on a second page of results. Will address this now with updated list of controls